src/CommunityToolkit.Maui.Core/Handlers/CameraView/CameraViewHandler.ios.cs

```diff
@@ -0,0 +1,82 @@
+﻿using Microsoft.Maui.Handlers;
+using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Maui.Core.Views.CameraView;
+
+namespace CommunityToolkit.Maui.Core.Handlers;
+
+public partial class CameraViewHandler : ViewHandler<ICameraView, UIView>, IDisposable
+{
+	public static Action<byte[]>? Picture { get; set; }
+	
+	CameraManager? cameraManager;
+
+	public static IPropertyMapper<ICameraView, CameraViewHandler> Propertymapper = new PropertyMapper<ICameraView, CameraViewHandler>(ViewMapper)
+	{
+		[nameof(ICameraView.CameraFlashMode)] = MapCameraFlashMode,
+		[nameof(IAvailability.IsAvailable)] = MapIsAvailable
+	};
```

Comment:
This could live in a shared class since it will be the same as Android and other platforms. I put everything on `CameraViewHandler.android.cs` file for simplicity, since I just have one platform implemented.